design/codebases.go

```diff
@@ -63,15 +64,29 @@ var workspace = a.Type("Workspace", func() {
 
 var workspaceAttributes = a.Type("WorkspaceAttributes", func() {
 	a.Description(`JSONAPI store for all the "attributes" of a workspace. +See also see http://jsonapi.org/format/#document-resource-object-attributes`)
+	a.Attribute("id", d.String, "The workspace id", func() {
+		a.Example("workspacephly8l91tqg0u08x")
+	})
 	a.Attribute("name", d.String, "The workspace name", func() {
 		a.Example("test")
 	})
-	a.Attribute("description", d.String, "The URL of the codebase ", func() {
+	a.Attribute("description", d.String, "The workspace description", func() {
 		a.Example("")
 	})
 	a.Attribute("status", d.String, "The workspace status", func() {
 		a.Example("STOPPED");
 	})
+	a.Attribute("links", links, "The workspace links")
```

Comment:
links should be on workspace, not workspaceattributes

Reply:
Could you clarify why because in workspace model `id  / description / status / link` are on the same level:
```
{
  "config": {
    "commands": [
      {
        "attributes": {
          "goal": "string",
          "previewUrl": "string"
        },
        "commandLine": "string",
        "name": "string",
        "type": "string"
      }
    ],
    "defaultEnv": "string",
    "description": "string",
    "environments": {
      "default": {
        "machines": {},
        "recipe": {
          "content": "string",
          "contentType": "string",
          "location": "string",
          "type": "string"
        }
      }
    },
    "links": [
      {
        "href": "string",
        "method": "string",
        "rel": "string"
      }
    ],
    "name": "string",
    "projects": [
      {
        "attributes": {},
        "description": "string",
        "links": [
          {}
        ],
        "mixins": [
          "string"
        ],
        "name": "string",
        "path": "string",
        "source": {
          "location": "string",
          "parameters": {},
          "type": "string"
        },
        "type": "string"
      }
    ]
  },
  "id": "string",
  "links": [
    {
      "href": "string",
      "method": "string",
      "rel": "string"
    }
  ],
  "runtime": {
    "devMachine": {
      "runtime": {
        "servers": {}
      }
    }
  },
  "status": "string"
}
```